eisbuk-admin/src/App.tsx

```diff
@@ -37,4 +40,22 @@ const App: React.FC = () => {
   );
 };
 
+// ***** Region Styles ***** //
+const useStyles = makeStyles((theme) => ({
```

Comment:
I've removed the Theme typing altogether as, at most we don't need it, so it's best to have the `makeStyles` type it implicitly.